homeassistant/components/microbees/entity.py

```diff
@@ -0,0 +1,52 @@
+"""Base entity for microBees."""
+
+from microBeesPy.microbees import Actuator, Bee
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+
+
+class MicroBeesEntity(CoordinatorEntity[MicroBeesUpdateCoordinator]):
+    """Base class for microBees entities."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        actuator_id: int,
+        bee_id: int,
+        coordinator: MicroBeesUpdateCoordinator,
```

Comment:
```suggestion
        coordinator: MicroBeesUpdateCoordinator,
        bee_id: int,
        actuator_id: int,
```

This seems way more logical as the coordinator is an essential part, and an actuator is part of a bee